lib/SILOptimizer/PassManager/PassManager.cpp

```diff
@@ -772,6 +772,10 @@ void SILPassManager::runPassOnFunction(unsigned TransIdx, SILFunction *F) {
       runSwiftFunctionVerification(F);
     }
   }
+  if (getOptions().VerifyOwnershipAll &&
```

Comment:
nitpick: shouldn't this be in an else-branch, so that ownership verification is not done twice if both, `-sil-verify-all` and `-sil-ownership-verify-all` is specified?